src/models/d2Model.ts

```diff
@@ -59,7 +59,7 @@ export abstract class D2Model {
         const filter = _.compact([
             search && isValidUid(search) ? `id:eq:${search}` : null,
             search && !isValidUid(search) ? `displayName:ilike:${search}` : null,
-            lastUpdatedDate ? `lastUpdated:ge:${lastUpdatedDate.toISOString()}` : null,
+            lastUpdatedDate ? `lastUpdated:ge:${lastUpdatedDate.format("YYYY-MM-DD")}` : null,
```

Comment:
Maybe this?

Reply:
Not sure about this @SferaDev I guess the equivalent code is https://github.com/EyeSeeTea/metadata-synchronization/blob/development/src/models/deletedObjects.ts#L79 but not sure if there is anything we would like to migrate. Can you shed some light on this?

Reply:
We don't have any ``toISOString()`` anymore, so it's safe to asume we already patched this (lastUpdated using time/date instead of only date)

Reply:
Great! Thanks!